Lib/test/support/__init__.py

```diff
@@ -155,8 +155,12 @@ def get_attribute(obj, name):
     try:
         attribute = getattr(obj, name)
     except AttributeError:
-        raise unittest.SkipTest("module %s has no attribute %s" % (
-            obj.__name__, name))
+        if isinstance(obj, type(sys)):
```

Comment:
types is already imported by test.support through functools ;-) Again, it was just a comment, it's unrelated to the backport itself.

```
haypo@selma$ ./python -i -S
Python 3.7.0a0 (heads/shutdown_10022:1315e80, May 10 2017, 18:49:18) 
[GCC 6.3.1 20161221 (Red Hat 6.3.1-1)] on linux
>>> import test
>>> import test.support
>>> import sys
>>> 'types' in sys.modules
True
```